lib/server.js

```diff
@@ -189,3 +203,21 @@ module.exports = {
   }
 };
 
+function cleanUpAndExit() {
+  if (electronPids) {
```

Comment:
Since I was able to reproduce with ``client.create()`` in ``renderer process`` only, I was verifying your fix there. Noticed that when I patched ``server.js`` with your fix, it was terminating the process gracefully when I was expecting it to force-kill with ``SIGKILL``. So, I kept pulling out pieces of code and verifying at each step. Finally, found that just the following code (no other changes) was able to terminate the process gracefully.
```js
function cleanUpAndExit() {
  console.log('=====>>>>> cleanUpAndExit');
  setTimeout(function() {
    process.exit(0);
  }, 1000);
}
process.on('SIGINT', cleanUpAndExit);
process.on('SIGTERM', cleanUpAndExit);
```
Not sure how this made the difference. Can you check in your setup?

And, btw, I'm using ``electron-prebuilt 1.2.2``. What version of electron you are using in your app?